database/fixtures.sql

```diff
@@ -3,15 +3,76 @@ BEGIN;
+
+INSERT INTO digitalprint (magicsPath) VALUES ('path1'), ('path2');
+
+INSERT INTO magicspairing(digitalPrintID, digitalPartID, label) VALUES
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Screw'), 'Screw1'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Screw'), 'Screw2'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Screw'), 'Screw3'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Screw'), 'Screw4'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Screw'), 'Screw5'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Hinge'), 'Hinge1'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Hinge'), 'Hinge2'),
+  ((select id from digitalprint where magicsPath = 'path1'), (select id from digitalpart where name = 'Hinge'), 'Hinge3');
+
+
+INSERT INTO magicspairing(digitalPrintID, digitalPartID, label) VALUES
+  ((select id from digitalprint where magicsPath = 'path2'), (select id from digitalpart where name = 'Emil'), 'EmilProtype'),
+  ((select id from digitalprint where magicsPath = 'path2'), (select id from digitalpart where name = 'Emilia'), 'EmiliaProtype');
+
+INSERT INTO physicalprint (digitalPrintID, slmPath) VALUES
+  ((select id from digitalprint where magicsPath = 'path1'), 'slm/path1'),
+  ((select id from digitalprint where magicsPath = 'path1'), 'slm/path2');
+
+INSERT INTO physicalprint (digitalPrintID, slmPath) VALUES
+  ((select id from digitalprint where magicsPath = 'path2'), 'slm/path3'),
+  ((select id from digitalprint where magicsPath = 'path2'), 'slm/path4'),
+  ((select id from digitalprint where magicsPath = 'path2'), 'slm/path5'),
+  ((select id from digitalprint where magicsPath = 'path2'), 'slm/path6'),
+  ((select id from digitalprint where magicsPath = 'path2'), 'slm/path7');
+
+
+
+INSERT INTO orderedpart (orderID, digitalPartID, amount) VALUES
+  ((select id from "order" where date = 2017-10-10), (select id from digitalpart where name = 'Screw'), 40),
+  ((select id from "order" where date = 2017-11-01), (select id from digitalpart where name = 'Hinge'), 10);
+
+INSERT INTO orderedpart (orderID, digitalPartID, amount) VALUES
+  ((select id from "order" where date = 2017-09-09), (select id from digitalpart where name = 'Emil'), 6),
+  ((select id from "order" where date = 2017-09-09), (select id from digitalpart where name = 'Emilia'), 6);
+
+INSERT INTO physicalpart (physicalPrintID, orderedPartID, magicsPartPairingID) VALUES
+  (1,1,1),(1,1,2),(1,1,3),(1,1,4),(1,1,5),(1,2,6),(1,2,7),(1,2,8),
+  (2,1,1),(2,1,2),(2,1,3),(2,1,4),(2,2,6); /*Skipping three on purpose*/
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This single line comment should use the single line comment syntax "--" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=plsql%3ASingleLineCommentsSyntaxCheck)